plaid/holdings.go

```diff
@@ -0,0 +1,73 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Security struct {
+	SecurityID             string  `json:"security_id"`
+	CUSIP                  string  `json:"cusip"`
+	SEDOL                  string  `json:"sedol"`
+	ISIN                   string  `json:"isin"`
+	InstitutionSecurityID  string  `json:"institution_security_id"`
+	InstitutionID          string  `json:"institution_id"`
+	ProxySecurityID        string  `json:"proxy_security_id"`
+	Name                   string  `json:"name"`
+	TickerSymbol           string  `json:"ticker_symbol"`
+	IsCashEquivalent       bool    `json:"is_cash_equivalent"`
+	Type                   string  `json:"type"`
+	ClosePrice             float64 `json:"close_price"`
+	ClosePriceAsOf         string  `json:"close_price_as_of"`
+	ISOCurrencyCode        string  `json:"iso_currency_code"`
+	UnofficialCurrencyCode string  `json:"unofficial_currency_code"`
```

Comment:
Why do we have `UnofficialCurrencyCode` at all? Developers might find that confusing

Reply:
We've documented this in our docs that we've sent out to customers. This is primarily intended for securities like cryto (BTC) or rewards points at certain institutions. Is this something you think we should remove?